### DISCUSSION TITLE
人工智慧

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1787] [General]
人工智慧的發展由艾倫·圖靈於1950年發表《運算機器與智慧》（Computing Machinery and Intelligence）的論文裡提問：「機器能思考嗎？（Can Machine Think？）」，
其中所提出的圖靈測試（Turing Test），主要在判斷機器是否具備智慧與人類進行對話，如果機器可以與人類展開對話並且不被辨別出其機器身份，那就算通過測試。故圖靈測試已成為測試機器是否具備智慧的準則。
「人工智慧」在早期的定義是由約翰·麥卡錫於1955年提出，就是要讓機器的行為看起來就像是人類的行為，表現出與人類思考模式一樣的表現，目的在讓電腦可以具有像人類思考、解決問題的能力。

資料來源：維基百科（讀者自我整理）

主要在判斷機器是否具備智慧與人類進行對話，如果機器可以與人類展開對話並且不被辨別出其機器身份，那就算通過測試
對於圖靈測試這個評斷的標準是什麼？
像是平常跟ChatGPT或是SIRI講話，大多都可以感覺得到是機器人而不是人類的口語，所以對於不被辨別出機器身份的標準是什麼～

**Comment by asd99263fg** [comment]
人工智慧（AI）是指模擬人類智能的技術，使機器能夠執行諸如學習、推理、問題解決和決策等任務。AI的優勢在於能夠處理和分析大量數據，從中提取有價值的資訊，提高工作效率和精確度，並能自動化繁瑣或重複的任務。此外，AI可以應用於醫療診斷、金融風險評估、個性化推薦和自動駕駛等多個領域，從而改進服務質量並促進創新和發展。
